src/UI/Series/Details/InfoViewTemplate.hbs

```diff
@@ -55,6 +55,10 @@
             {{#if_eq seasonNumber compare="-1"}}
                 <span class="label label-default">{{title}}</span>
             {{/if_eq}}
+
+            {{#if_eq sceneSeasonNumber compare="-1"}}
```

Comment:
I thought that with this change you would also support mapping Spartacus properly. So seasonNumber=0 and sceneSeasonNumber=1. Would've been nice and a welcome change.